docs/runbooks/CnaoDown.md

```diff
@@ -2,29 +2,35 @@
 
 ## Meaning
 
-Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
-This alert fires when CNAO operator is down.
+The `Cluster-network-addons-operator` (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when the CNAO is down.
 
 ## Impact
 
-With CNAO operator down, changes in the components are not reconciled, and as a result might not be properly deployed.
+If the CNAO is not running, the cluster cannot reconcile changes to virtual machine components. As a result, the changes might fail to take effect.
 
 ## Diagnosis
 
-- Check CNAO's operator pod namespace:
+1. Obtain the namespace of the CNAO pod:
 	```
-	export NAMESPACE="$(kubectl get deployment -A | grep cluster-network-addons-operator | awk '{print $1}')"
+	$ export NAMESPACE="$(kubectl get deployment -A | grep cluster-network-addons-operator | awk '{print $1}')"
 	```
 
-- Check to see if CNAO's operator pod is down.
+1. Check the status of the CNAO pod:
```

Comment:
I think the pod name should be written in full in a procedure step: "`cluster-network-addons-operator` pod"